docker/machine_learning/stable_diffusion_webui/Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM alpine/git:2.36.2 as download
+
+COPY clone.sh /clone.sh
+
+RUN . /clone.sh taming-transformers https://github.com/CompVis/taming-transformers.git 24268930bf1dce879235a7fddd0b2355b84d7ea6 \
+  && rm -rf data assets **/*.ipynb
+
+RUN . /clone.sh stable-diffusion-stability-ai https://github.com/Stability-AI/stablediffusion.git 47b6b607fdd31875c9279cd2f4f16b92e4ea958e \
+  && rm -rf assets data/**/*.png data/**/*.jpg data/**/*.gif
+
+RUN . /clone.sh CodeFormer https://github.com/sczhou/CodeFormer.git c5b4593074ba6214284d6acd5f1719b6c5d739af \
```

Comment:
I feel like these are secrets that we shouldn't be committing. Please correct me if I'm wrong

Reply:
> I feel like these are secrets that we shouldn't be committing. Please correct me if I'm wrong

These are the git commit hashes

Reply:
Got it thanks.